agot-bg-game-server/src/common/ingame-game-state/planning-game-state/PlanningGameState.ts

```diff
@@ -208,6 +208,11 @@ export default class PlanningGameState extends GameState<IngameGameState> {
         return this.readyPlayers.includes(player);
     }
 
+    ordersAssignedToAllPossibleRegions(house: House) :boolean {
```

Comment:
It should be `): boolean` (the space after the double-dot).

Also the name of the function could follow the convention of the rest of the functions with something like `areOrdersAssignedToAllPossibleRegions`.